quinn-proto/src/lib.rs

```diff
@@ -131,6 +131,12 @@ pub enum Directionality {
     Uni = 1,
 }
 
+impl Directionality {
+    pub(crate) fn iter() -> impl Iterator<Item = Self> {
```

Comment:
Does this even need to be `pub(crate)`, since it's defined in the root module?

Reply:
Oh, you're right!